src/main/scala/org/aphreet/c3/snippet/groups/snippet/GroupPageFiles.scala

```diff
@@ -743,6 +743,7 @@ class GroupPageFiles(data: GroupPageFilesData) extends C3ResourceHelpers
           TAGS_META -> tags.trim,
           DESCRIPTION_META -> description.trim,
           ACL_META -> currentDirectory.metadata.get(ACL_META).getOrElse(""))
+        name = name.replace(':', '-').replace('/', '-').replace('\\', '-');
```

Comment:
I suppose that need create separate function for name normalization, as the same time necessary to save original name in metadata.